 01-fix-scripts.diff

 findpy not available in /usr/bin + use /usr/bin/python. Patch by <alf@merlin>.

--- codespeak-lib-0.7-svn20061228.orig/py/misc/cmdline/countloc.py      2006-12-28 14:14:32.000000000 +0100
+++ codespeak-lib-0.7-svn20061228/py/misc/cmdline/countloc.py   2007-01-05 17:23:04.000000000 +0100
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # hands on script to compute the non-empty Lines of Code 
 # for tests and non-test code 
